### DISCUSSION TITLE
我来试试怎么发帖

### DISCUSSION THREAD

**Discussion by pwshanjiji** [share121/inter-knot#5240] [灌水]
是这样吗
![1724175083250](https://github.com/user-attachments/assets/a46b5593-c647-4f6c-af16-f086788fbdf3)
![1724563868519](https://github.com/user-attachments/assets/dd6939c7-386d-4230-9b28-f27f1359cdfd)
图片

**Comment by Kazuha-hhh** [comment]
看不到图片哦
